source/_integrations/hydrawise.markdown

```diff
@@ -27,114 +28,36 @@ There is currently support for the following device types within Home Assistant:
 - [Sensor](#sensor)
 - [Switch](#switch)
 
-## Configuration
+## Prerequisites
 
-To enable it, add the following to your `configuration.yaml` file:
+To set up the Hydrawise integration, you must first obtain an API Key.
 
-```yaml
-# Example configuration.yaml entry
-hydrawise:
-  access_token: YOUR_API_KEY
-```
+1. Log in at [https://app.hydrawise.com](https://app.hydrawise.com).
+2. Go to 'Account Details' under the 'My Account' menu (in the upper-right hand corner).
+3. Under the **Account Settings** section, copy the **API Key**.
+   1. If there is not an API Key present, click the **Generate API Key** button.
 
-{% configuration %}
-access_token:
-  description: The API KEY assigned to your Hydrawise account.
-  required: true
-  type: string
-scan_interval:
-  description: The time interval, in seconds, to poll the Hydrawise cloud.
-  required: false
-  type: integer
-  default: 120
-{% endconfiguration %}
-
-To get your API access token log into your [Hydrawise account](https://app.hydrawise.com/config/login) and in the 'My Account Details' section under Account Settings click 'Generate API Key'. Enter that key in your configuration file as `YOUR_API_KEY`.
+{% include integrations/config_flow.md %}
 
 ## Binary Sensor
 
-Once you have enabled the `hydrawise` integration, add the following to your `configuration.yaml` file:
-
-```yaml
-# Example configuration.yaml entry
-binary_sensor:
-  - platform: hydrawise
-```
-
-{% configuration %}
-monitored_conditions:
-  description: The binary sensors that should be displayed on the frontend.
-  required: false
-  type: list
-  default: All binary sensors are enabled.
-  keys:
-    is_watering:
-      description: The binary sensor is `on` when the zone is actively watering.
-    status:
-      description: This will indicate `on` when there is a connection to the Hydrawise cloud. It is not an indication of whether the irrigation controller hardware is online.
-{% endconfiguration %}
-
-<div class='note warning'>
```

Comment:
Should these warnings really be removed?

Reply:
I added the first warning back, but the second one isn't relevant since config flow doesn't support modifying the scan interval.